### ISSUE TITLE
Crate fails to compile on Windows due to trying to import non-existent FILE_MODE

### ISSUE THREAD

**Issue by str4d** [tendermint/signatory#121]
```
   Compiling signatory v0.9.2
error[E0432]: unresolved import `super::FILE_MODE`
 --> C:\Users\me\.cargo\registry\src\github.com-1ecc6299db9ec823\signatory-0.9.2\src\encoding\encode.rs:8:5
  |
8 | use super::FILE_MODE;
  |     ^^^^^^^^^^^^^^^^ no `FILE_MODE` in `encoding`

error[E0432]: unresolved import `super::FILE_MODE`
  --> C:\Users\me\.cargo\registry\src\github.com-1ecc6299db9ec823\signatory-0.9.2\src\encoding\pkcs8.rs:20:5
   |
20 | use super::FILE_MODE;
   |     ^^^^^^^^^^^^^^^^ no `FILE_MODE` in `encoding`

error: aborting due to 2 previous errors
```

`FILE_MODE` is [only included for Unix](https://github.com/tendermint/signatory/blob/master/src/encoding/mod.rs#L24-L26), but is [imported if the std feature is enabled](https://github.com/tendermint/signatory/blob/master/src/encoding/pkcs8.rs#L19-L20), even though [it is only used for Unix](https://github.com/tendermint/signatory/blob/master/src/encoding/pkcs8.rs#L60-L73).

**Comment by tony-iqlusion** [comment]
This should hopefully be addressed by #122

**Comment by tony-iqlusion** [comment]
Released in v0.9.3 (#124)

**Comment by str4d** [comment]
Still fails:

```
   Compiling signatory v0.9.3
error[E0432]: unresolved import `super::FILE_MODE`
  --> C:\Users\me\.cargo\registry\src\github.com-1ecc6299db9ec823\signatory-0.9.3\src\encoding\pkcs8.rs:20:5
   |
20 | use super::FILE_MODE;
   |     ^^^^^^^^^^^^^^^^ no `FILE_MODE` in `encoding`

error: aborting due to previous error
```

**Comment by tarcieri** [comment]
@str4d can you check if `master` is fixed after #125?

**Comment by str4d** [comment]
Yep, current master (f446c72b) fixes compilation on Windows.

**Comment by tony-iqlusion** [comment]
Released as v0.9.4 (#126)
